src/Framework/AbstractConfigGacela.php

```diff
@@ -34,9 +15,11 @@ public function config(): array
     }
 
     /**
+     * @param array<string,mixed> $globalServices
+     *
      * @return array<string,string|callable>
      */
-    public function mappingInterfaces(): array
+    public function mappingInterfaces(array $globalServices): array
     {
         return [];
```

Comment:
```suggestion
    public function mappingInterfaces(array $globalServices): array
    {
        return $globalServices;
```

What do you think @Chemaclass?

Reply:
Good eye, but it's not what we want. The global-services are *not* the same as the mapping-interfaces. I updated the signature code to reflect this intention clearer.
```php
/**
 * @param array<string,mixed> $globalServices
 *
 * @return array<class-string,class-string|callable>
 */
public function mappingInterfaces(array $globalServices): array
{
    return [];
}
```

Reply:
That's true.